command/cp.go

```diff
@@ -251,8 +257,8 @@ type Copy struct {
 	acl                   string
 	forceGlacierTransfer  bool
 	ignoreGlacierWarnings bool
-	exclude               []string
 	raw                   bool
+	exclude               []string
```

Comment:
Yes and no. In first draft (commit) I was going to add a field to Copy object. While I add `retryOnNoSuchUploadError` field to Copy, I thought it was weird a `string slice` is placed between `bool` fields. So I moved `exclude` field to 1 line below. Then, after I've removed `retryOnNoSuchUploadError` field from Copy object, I didn't reverted the place of `exclude` field since it was minor, and at worst harmless change.